src/pages/PaymentConfirm/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
+import Lottie from 'lottie-react';
+import { useSearchParams } from 'react-router-dom';
+import useConfirmPayments from '../Payment/hooks/useConfirmPayments';
+import Paying from '@/assets/Payment/paying.json';
+
+export default function PaymentConfirm() {
+  const [searchParams] = useSearchParams();
+  const orderType = searchParams.get('orderType');
+  const orderId = searchParams.get('orderId');
+  const paymentKey = searchParams.get('paymentKey');
+  const amount = searchParams.get('amount');
```

Comment:
type-safe하게 처리해야 할 거 같아요. UI가 별도로 정해져 있지 않아서 직접 추가하기 힘들지만 우선은 조건문으로 처리해놓으면 좋을 거 같습니다.
```suggestion
  const orderType = searchParams.get('orderType');
  const orderId = searchParams.get('orderId');
  const paymentKey = searchParams.get('paymentKey');
  const amount = searchParams.get('amount');
  
  if(!orderType || !orderId || !paymentKey || !amount) {
    // UI 처리 or 에러 처리
  }
```

Reply:
말씀해주신대로 UI가 정해져있지 않아서 우선 단순 문구 보여주는 식으로 작성해두었습니다.
상황을 생각해봤을때 토스에서 결제는 진행되었는데 백엔드 검증은 실패하게 될 것 같아 다음 스프린트 때 결제 취소 후 리다이렉트 혹은 에러페이지를 추가하도록 논의해보도록 하겠습니다

[102b42f](https://github.com/BCSDLab/KOIN_ORDER_WEBVIEW/pull/132/commits/102b42f3d668e80b9d3fd4b5766d201fa9ae49c1)